src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -5,31 +5,37 @@
       <i class="fa fa-bookmark pull-left "></i>
     </div>
-    <div class=" workItemQuickAdd_story">
-      <div class="col-md-6 col-sm-6 col-xs-12 workItemQuickAdd_storyInput">
-        <input #quickAddTitle type="text" id="exampleInput" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.title']" 
-              (keyup.enter)="save($event)"
-              (keyup)="checkTitle()" placeholder="A New Story" required/>
-      </div>
-      <div class="col-md-6 col-sm-6 mobdn">
-        <textarea #quickAddDesc rows="1" type="text" id="exampleDesc" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.description']" 
-                [style.height]="descHeight"
-                [style.resize]="descResize"
-                (keydown.enter)="save($event)"
-                (keyup.enter)="preventDef($event)"
-                (keyup)="checkDesc()"
-                placeholder="Description">
-        </textarea>
-      </div>
-    </div>
-    <div class="workItemQuickAdd_AddBlk">
-      <span class="pull-left font16 workItemQuickAdd_Add" 
-        (click)="save($event)" [ngClass]="{'icon-btn': validTitle,'icon-btn-disabled':!validTitle}"><i class="fa fa-check font14"></i></span>
-      <span (click)="toggleQuickAdd()" class="pull-right closeQuickAdd"><i class="pficon pficon-close font16"></i></span>
+    <div [hidden]="submitted">
+      <form (ngSubmit)="save()" #almWorkItemQuickAddForm="ngForm"
+        id="wi-quck-add-form">
+        <div class="workItemQuickAdd_story">
+            <div class="col-md-6 col-sm-6 col-xs-12 workItemQuickAdd_storyInput">
+              <input #quickAddTitle type="text" id="exampleInput" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.title']" 
+                    (keyup.enter)="save($event)"
+                    (keyup)="checkTitle()" placeholder="A New Story" name="title" required/>
+            </div>
+            <div class="col-md-6 col-sm-6 mobdn">
+              <textarea #quickAddDesc rows="1" name="description" type="text" id="exampleDesc" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.description']" 
+                      [style.height]="descHeight"
+                      [style.resize]="descResize"
+                      (keydown.enter)="save($event)"
+                      (keyup.enter)="preventDef($event)"
+                      (keyup)="checkDesc()"
+                      placeholder="Description">
+              </textarea>
+            </div>
+        </div>
+        <div class="workItemQuickAdd_AddBlk">
+          <button type="submit" [disabled]="!validTitle" [ngClass]="{'icon-btn': validTitle,'icon-btn-disabled':!validTitle}" class="pull-left font16 workItemQuickAdd_Add">
+            <i class="fa fa-check font14"></i>
+          </button>
+          <button (click)="toggleQuickAdd()" class="pull-right closeQuickAdd"><i class="pficon pficon-close font16"></i></button>
```

Comment:
When I click on code, the url looks like http://0.0.0.0:8088/work-item-list?title=Test%2BTitle&description=Desc
the title and description are appended to the URL